Pixel Mapping

Sandeep-Sampath opened the pull request:
This PR is for;
- [ ] Mapping the pixels for 16-bit images.
  - Mapping the pixels for live image.
- [ ] Scaling16-bit images.
- [ ] This fixes #50, #51

Commit:
Image Averaging and Calibration Feature

This commit contains;
1. Image Averaging feature
1.1 Created an actor to perform averaging and to save the result in .tiff and .raw formats.
1.2 Messages to communicate with the controller are added.
1.3 Application Settings has been modified to select the path to save the averaged image.
1.4 Added methods to Launch Image Averaging.
2 Calibration
2.1 Calibration feature has been designed to access the feature from the dll file.
2.2 A class is created to perform the sensor calibration.
2.3 Added  Generate Reference file to the driver library.
2.4 Updated BontechWrapperLibrary.dll with the latest code
3. User-interface has been modified to incorporate the latest changes
4. Added messages to the User-Interface to communicate with the Controller.
5. Updated .lvlib, and .lvclass files to the latest changes.

This fixes #48 #42

Commit:
Pixel Mapping

This commit contains;
1. Developed a code to map the pixels for 16-bit images.
 1.1 Mapped the pixels to live image.
2. For saving 16-bit images pixel scaling has been added.

This fixes #50 #51

Commit:
Image Scaling Removed

This commit has disabled the image scaling to save 16-bit images in .tiff format

achuthaperumal reviewed (changes requested):
Add Option to Enable / Disable the Mapping Mode.

Commit:
Pixel Mapping feature

- [] Added a checkbox to enable/disable pixel mapping in UI.
- [] Added Messages, methods and codes for pixel mapping

Sandeep-Sampath commented:
- [x] Added a checkbox to enable/disable pixel mapping in UI.
- [x] Added Messages, methods and codes for pixel mapping

achuthaperumal reviewed (changes requested):
Remember to set `Bit Depth=0` before applying the mapping.

Sandeep-Sampath committed:
Pixel Mapping and LV22 Updation

This code contains;
1. Pixel mapping algorithm in live screen;
2. Image averaging has been reworked with averaging with an array but not images;
3. Mass compiled with LV22 from LV21.

Sandeep-Sampath committed:
Delete Find Max Min Value.vi

Deleted unwanted vi.
